src/praisonai-agents/praisonaiagents/session/api.py

```
@@ -589,6 +596,46 @@ def send_message(self, message: str, **kwargs) -> str:
+                    self._memory.close_connections()
+            except Exception as e:
+                logger.warning(f"Memory cleanup failed: {e}")
+            finally:
+                self._memory = None
+        
+        # Clear knowledge
+        if hasattr(self, '_knowledge') and self._knowledge:
+            try:
+                self._knowledge = None
+            except Exception:
+                pass  # Ignore cleanup errors
```

Ruff (0.15.15)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)